Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/ml_interface.hpp

```diff
@@ -4,9 +4,15 @@
 #include "librealsense2/rs.hpp"
 #include "../classification_result.hpp"
 
+/// <summary>
+/// This struct creates a model interface for object recognition models
```

Comment:
Is this comment necessary? Is it saying anything?
Isn't the do_work comment (perhaps expanded a little) enough?